packages/help-center/src/hooks/use-opening-coordinates.ts

```diff
@@ -31,8 +44,17 @@ export const calculateOpeningPosition = ( element: HTMLElement ) => {
 
 	const { x, y, width, height } = element.getBoundingClientRect();
 
-	const buttonLeftEdge = x;
-	const buttonRightEdge = x + width;
+	const position = getPosition( element );
```

Comment:
One small suggestion is being a little more specific with this variable. Position is kind of ambiguous. But maybe that's  me.

Reply:
Sure, using now `originElementOffset`